dtfs-central-api/src/types/db-models/banks.ts

```diff
@@ -3,7 +3,9 @@ import { WithId } from 'mongodb';
 type ReportPeriodSchedule = {
   startMonth: number;
   endMonth: number;
-}
+};
+
+export type BankReportPeriodSchedule = ReportPeriodSchedule[];
```

Comment:
We now refer to both singular and plural as a schedule. Maybe the singular should be something like `ReportPeriodMonthRange` instead `ReportPeriodSchedule`?

Reply:
Decided to remove `ReportPeriodSchedule` and just define
```
type BankReportPeriodSchedule = {
  startMonth: number;
  endMonth: number;
}[];
```